utilities/test_suite/rpp_test_suite_common.h

```diff
@@ -73,6 +73,7 @@ std::map<int, string> augmentationMap =
     {1, "gamma_correction"},
     {2, "blend"},
     {4, "contrast"},
+    {5, "pixelate"},
```

Comment:
The pixelate case number was not added in the supportedCaseList array in python
https://github.com/fiona-gladwin/rpp/blob/sn/pixelate_tensor/utilities/test_suite/HOST/runTests.py#L415

Please ensure to run QA tests and see everything is working as expected

Reply:
added case in supportedCaseList